api/types/installers/agentless-installer.sh.tmpl

```diff
@@ -12,6 +12,11 @@ set -o nounset
 
   . /etc/os-release
 
+  PACKAGE_LIST="{{ .TeleportPackage }}"
+  if [[ "{{ .AutomaticUpgrades }}" == "true" ]]; then
+    PACKAGE_LIST="${PACKAGE_LIST} {{ .TeleportPackage }}-updater"
```

Comment:
Don't we need to install _only_ `xxx-updater` package? I think it will pull teleport package as dependency?

@hugoShaka @fheinecke Am I wrong?

Reply:
It does :+1: 
https://apt.releases.development.teleport.dev/ubuntu/dists/bionic/stable/v13/binary-amd64/Packages
![image](https://user-images.githubusercontent.com/689271/235978582-5c60622b-3979-444f-949a-089985ff3ca2.png)

We can remove it, however I would like to clarify something first:
`teleport-ent-updater` depends on `teleport`
`teleport` and `teleport-ent` both provide `teleport`.

So, when we try to install only `teleport-ent-updater` which package is going to be pulled? Is it `teleport` or `teleport-ent`?

Reply:
Yes, but we don't want the `teleport-ent` package to be deleted if the users wants to uninstall the updater. IMO it makes sense to mark both as wanted, so `teleport-ent` won't be auto-removed.

Reply:
@hugoShaka In your documentation guide we instruct people to just `apt install teleport-ent-updater`. Is that for scenario when you already have an agent and want to add updater to it?

@fheinecke I think `teleport-ent-updater` should depend on `teleport-ent`, not `teleport`. Can we update that?

Reply:
>  Is that for scenario when you already have an agent and want to add updater to it?

Yes, the guide explains how to enroll an existing agent into automatic updates

Reply:
> I think teleport-ent-updater should depend on teleport-ent, not teleport. Can we update that?
@r0mant it has been (tested before I went on vacation, see https://github.com/gravitational/teleport.e/pull/1177)

Reply:
> we don't want the `teleport-ent` package to be deleted if the user wants to uninstall the updater.

I'll keep both packages then :+1: